src/main/java/de/samply/common/ldmclient/cql/LdmClientCql.java

```diff
@@ -158,6 +172,81 @@ private QueryResultStatistic createQueryResultStatistic(MeasureReport measureRep
     return queryResultStatistic;
   }
 
+  /**
+   * Create a sublist for an entity.
+   *
+   * @param location the measure location url
```

Comment:
This is simply the measure URI. It may come from a Location header, but it's just the URI.